src/main/java/nextstep/ladder/domain/GameResult.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.ladder.domain;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class GameResult {
+
+    private final Map<String, String> results;
+
+    public GameResult(Map<String, String> results) {
+        this.results = results;
+    }
+
+    public String getResultByParticipantName(String name) {
+        validateParticipant(name);
+        return results.get(name);
```

Comment:
```suggestion
        return Optional.ofNullable(results.get(name)).orElseThrow;
```
Optional을 사용해봐도 좋았을 것 같네요